test/unit/cpu/test-reduce.cpp

```diff
@@ -65,63 +65,73 @@ using namespace RAJA;
 
 
 using TestingTypes = ::testing::
-    Types<std::tuple<ExecPolicy<seq_segit, seq_exec>, seq_reduce>,
-          std::tuple<ExecPolicy<seq_segit, simd_exec>, seq_reduce>
+    Types<  // std::tuple<ExecPolicy<seq_segit, seq_exec>, seq_reduce>,
```

Comment:
Because they are both expensive and slow without testing anything noticeably different than the others.  If you prefer we can make it an ifdef for something like "RAJA_TEST_EXHAUSTIVE", but especially the tests with parallel for on the inner loop are needlessly expensive.